eng/docgeneration/Generate-DocIndex.ps1

```diff
@@ -109,7 +109,7 @@ function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang) {
     New-Item -Path $YmlPath -Name "toc.yml" -Force
     $visitedService = @{}
     # Sort and display toc service name by alphabetical order.
-    foreach ($serviceMapping in $tocContent.getEnumerator() | Sort Value) {
+    foreach ($serviceMapping in $tocContent.getEnumerator() | Sort Value | Sort Key) {
         $artifact = $serviceMapping.Key
         $serviceName = $serviceMapping.Value
```

Comment:
```suggestion
    foreach ($artifact in ($tocContent.Keys | Sort-Object)) {
        $serviceName = $tocContent[$artifact]
```
This will sort all the packages but which cause them to be sorted on the individual service name.